### PR TITLE
Update system test

### PR DESCRIPTION
**Pull request by chengzhuzhang**
Update system tests by new way of running and area mean time series test.

### CODE REVIEW

**Comment by chengzhuzhang** [comment]
@forsyth2 I just updated the system tests. This is a small change and won't effect the code. Just to  let you know that the system test will be back running with this PR:
```
python all_sets.py -d all_sets.cfg
```

File: acme_diags/run.py [hunk 1]
```diff
@@ -188,8 +188,8 @@ def _remove_attrs_with_default_values(self, param):
 
     def _find_attrs_with_none_default_values(self, param):
         """
-        In the param, remove any parameters that
-        have their default value.
+        In the param, find any parameters that
+        have none default value.
```
Comment (forsyth2) [inline acme_diags/run.py]: I'd say `default value of None` to match the casing of Python's `None`.

Comment (forsyth2) [inline acme_diags/run.py]: I'd say `default value of None` just to match Python's casing of `None`.

Comment (chengzhuzhang) [reply acme_diags/run.py]: Thanks for reviewing. It appears to be a misspelling. I updated the PR.

**Pull request merged**